firmware/thruna/BMS/CMakeLists.txt

```diff
@@ -50,6 +50,24 @@ list(REMOVE_ITEM GOOGLETEST_TEST_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/Test/Src/main
 set(GOOGLETEST_TEST_INCLUDE_DIRS
     "${CMAKE_CURRENT_SOURCE_DIR}/Test/Inc")
 
+    
+# INCLUDES and SOURCES from shared
+# Define the shared code that you want to use in this project
+
+
+set(SELECTED_SHARED_SRCS
+    ${ALL_SHARED_APP_SRCS}
+    ${ALL_SHARED_IO_SRCS}
+    ${ALL_SHARED_HW_SRCS}
+)
+
+set(SELECTED_SHARED_INCLUDE_DIRS
+    ${ALL_SHARED_APP_INCLUDE_DIRS}
+    ${ALL_SHARED_IO_INCLUDE_DIRS}
+    ${ALL_SHARED_HW_INCLUDE_DIRS}
+)
```

Comment:
Is this unnecessary - is there ever a case where a certain board will want to ignore  a certain include directory? 
I think every board will want to look in `app`, `io`, and `hw`. I believe headers only get included in the binary if a C file `#include`s it so this shouldn't affect the binary size either. Can you think if we will ever need this?

Reply:
I think you are right. It should not be a problem to include more directories for `target_include_directories` than needed. I will make this as a variable in `firmware/Cmakelist.txt`, and remove this argument from the functions.